entity-types/infra-kubernetes_cronjob/definition.yml

```diff
@@ -41,4 +41,11 @@ synthesis:
           present: false
         # value added for test entities only
         - attribute: newrelicOnly
-          value: "true"
\ No newline at end of file
+          value: "true"
+      tags:
+        k8s.cronjobName:
+          entityTagNames: [cronjob]
+        k8s.namespaceName:
+          entityTagNames: [namespace]
+        k8s.clusterName:
+          entityTagNames: [k8s.cluster.name]
```

Comment:
Why not use the same "tag names" that come from telemetry. Users tend to be confused when the names on telemetry and on tags are different from each other.

On the same matter, the golden tags definition have different tag names on them: https://github.com/newrelic/entity-definitions/pull/1468/files#diff-0c1214b8d257118b55b64d763c1e5d1a0e05b73da6c741e81f9f6f284d115964R11-R13

Reply:
I misunderstood the tag names and attribute names worked here, so the attribute sources and tag name were flipped. This should be resolved with the updates.